pango/Gir.toml

```diff
@@ -310,3 +313,36 @@ boxed_inline = true
     [[object.function]]
     name = "transform_rectangle"
     manual = true
+
+[[object]]
+name = "Pango.TabAlign"
+status = "generate"
+    [[object.member]]
+    name = "right"
+    version = "1.50"
+    [[object.member]]
+    name = "center"
+    version = "1.50"
+    [[object.member]]
+    name = "decimal"
+    version = "1.50"
```

Comment:
You created an issue/MR to mark these enums correctly in C?

Reply:
Well, as there's no proper annotations for enum members; they are kind of documented already https://docs.gtk.org/Pango/enum.Variant.html

Reply:
There is, see https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/230 . We use that extensively in GStreamer.

Reply:
Oh wow, didn't knew that. So why we always have to set them manually so far? :/

Reply:
Because gobject-introspection can only do that since a year :)

Reply:
https://gitlab.gnome.org/GNOME/pango/-/merge_requests/575